src/client/src/services/trades/__mocks__/trades.mock.ts

```diff
@@ -45,7 +45,7 @@ export const mockRawTrades = {
 
 export const mockTrades: FxTrade[] = [
   {
-    tradeId: "1111111111",
+    tradeId: "1111111111" as unknown as bigint,
```

Comment:
eww, we need to sort out those types properly .. as I say, though, we don't have to refactor everything at once